engines/router/missionctl/server/application.go

```diff
@@ -67,26 +67,31 @@ func Run() {
 		}
 
 		s := upi.NewUPIServer(missionCtl)
-		if cfg.AppConfig.CustomMetrics {
-			m := cmux.New(l)
-			grpcL := m.Match(cmux.HTTP2HeaderField("content-type", "application/grpc"))
-			httpL := m.Match(cmux.HTTP1Fast())
+		m := cmux.New(l)
+		grpcL := m.Match(cmux.HTTP2HeaderField("content-type", "application/grpc"))
+		httpL := m.Match(cmux.Any())
 
-			mux := http.NewServeMux()
+		mux := http.NewServeMux()
+		mux.Handle("/v1/internal/", http.StripPrefix(
+			"/v1/internal",
+			handlers.NewInternalAPIHandler([]string{
```

Comment:
Why has this been added?

Reply:
oops my bad, will remove the enricher/ensembler uri

"/v1/internal/" health handler was added for queue-proxy probe